### ISSUE TITLE
Access API is returning other applications and not only the requested 

### ISSUE THREAD

**Issue by josejulio** [RedHatInsights/insights-rbac#264]
**Describe the bug**
The `/access/` endpoint is not doing an exact match with the `application` param, we noticed this when we decided to rename from `custom-policies` to `policies`. We noticed that the call:
`/api/rbac/v1/access/?application=policies` still returned the `custom-policies` permission.

**To Reproduce**
1. Go to https://ci.cloud.redhat.com/ and login
2. Go to https://ci.cloud.redhat.com/api/rbac/v1/access/?application=po
3. See that it returns every permission of the application that have `po` on their name

```
{
    "meta": {
        "count": 7,
        "limit": 1000,
        "offset": 0
    },
    "links": {
        "first": "/api/rbac/v1/access/?application=po&limit=1000&offset=0",
        "next": null,
        "previous": null,
        "last": "/api/rbac/v1/access/?application=po&limit=1000&offset=0"
    },
    "data": [
        {
            "permission": "catalog:portfolios:read",
            "resourceDefinitions": [
                {
                    "attributeFilter": {
                        "key": "scope",
                        "value": "group",
                        "operation": "equal"
                    }
                }
            ]
        },
        ...
    ]
}
```

**Expected behavior**
I would expect the `application` to be an exact match of the application that we require, as show in the [openapi.json](https://ci.cloud.redhat.com/api/rbac/v1/openapi.json) doc

> The application name to obtain access for the principal

**Context (please complete the following information):**
 - OS: Fedora 
 - Browser: Chrome
 - Commit Hash from status endpoint: `c107d88022cf94b7744da5d35ec7ca98e0f25f75`

**Comment by coderbydesign** [comment]
Hi @josejulio - we actually have [a bug](https://projects.engineering.redhat.com/browse/RHCLOUD-5269) logged for this currently, and will be addressing it soon. I'll update this issue once a PR is in. Thank you!

**Comment by petracihalova** [comment]
closing the issue - fixed by #266
